docs/_includes/customizer-variables.html

```diff
@@ -1239,7 +1251,7 @@ <h2 id="forms">Forms</h2>
   </div>
   <div class="col-xs-4">
     <label for="input-@control-label-focus">@control-label-focus</label>
-    <input id="input-@control-label-focus" type="text" value="@brand-light-blue" data-var="@control-label-focus" class="form-control"/>
+    <input id="input-@control-label-focus" type="text" value="@brand-light-blue-minus-20" data-var="@control-label-focus" class="form-control"/>
```

Comment:
Why is the value here still a value and not a hardcoded variable like before?

Reply:
It's because some of the variables reference values, others reference other variables.  Each variable gets auto generated a form control.  So it's not a problem.